### ISSUE TITLE
Asthma Disease Detection Using Deep Learning

### ISSUE THREAD

**Issue by Swish78** [abhisheks008/ML-Crate#673]
### ML-Crate Repository (Proposing new issue)

:red_circle: **Project Title** : Asthma Disease Detection Using Deep Learning

:red_circle: **Aim** : Develop a robust model to detect asthma using deep learning techniques, specifically ANNs from PyTorch and custom-built ANNs using NumPy and SciPy.

:red_circle: **Dataset** : [Asthma Disease Dataset](https://www.kaggle.com/datasets/rabieelkharoua/asthma-disease-dataset)

:red_circle: **Approach** : 
1. Perform exploratory data analysis (EDA) to understand the dataset and uncover initial insights.
2. Implement data preprocessing techniques to handle outliers, feature engineering, and missing values.
3. Develop and compare multiple models:
   - ANN using PyTorch
   - Custom-built ANN using NumPy and SciPy
4. Evaluate model performance using metrics like Accuracy, Precision, Recall, F1-Score, and ROC-AUC.
5. Document the process, results, and conclusions thoroughly.

**********************************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Swayam 
- GitHub Profile Link : [Swayam](https://github.com/Swish78)
- Participant ID (If not, then put NA) : 
- Approach for this Project : Implementing a robust data preprocessing pipeline, experimenting with different deep learning architectures, and evaluating model performance comprehensively.
- What is your participant role? Contributor (VSOC'24)

I want to get this assigned.

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Can you implement 3-4 models for this project?

**Comment by abhisheks008** [comment]
Implement 2 ANNs and 5 ML Models for this problem statement.

Assigning this issue to you @Swish78

**Comment by Sanjanah8** [comment]
Issue Title: 673 Asthma Disease Detection Using Deep Learning

Full name : Sanjanah A
GitHub Profile Link : Sanjanah8(https://github.com/sanjanah8)
Participant ID (If not, then put NA) :
Approach for this Project : Implementing a robust data preprocessing pipeline, experimenting with different deep learning architectures, and evaluating model performance comprehensively.
What is your participant role? Contributor (SSOC'3 2024)

I have used ANN using pytorch and 5 different ML models for this approach. I want to get this assigned. Please review my pull request and any changes to be made can be rectified by me.

**Comment by abhisheks008** [comment]
> Issue Title: 713 Asthma Disease Detection Using Deep Learning
> 
> Full name : Sanjanah A GitHub Profile Link : Sanjanah8(https://github.com/sanjanah8) Participant ID (If not, then put NA) : Approach for this Project : Implementing a robust data preprocessing pipeline, experimenting with different deep learning architectures, and evaluating model performance comprehensively. What is your participant role? Contributor (SSOC'3 2024)
> 
> I have used ANN using pytorch and 5 different ML models for this approach. I want to get this assigned. Please review my pull request and any changes to be made can be rectified by me.

Please wait for the issue to be assigned to you, then only you start working on it. Without assigning an issue how can you start pushing your code?

What are the ML models you think are compatible with this dataset? As you need to implement 6-7 models for any issue here.

**Comment by adi271001** [comment]
Full name : Aditya D
GitHub Profile Link : https://github.com/Swish78
Participant ID (If not, then put NA) : NA
Approach for this Project :  using ml algos like logistic regression , knn , random forest , decision tree , svm , adaboost , catboost , gradientboost , xgboost and stacked classifier 
What is your participant role? SSOC Contributor   @abhisheks008 if this issue is still open please assign it to me

**Comment by abhisheks008** [comment]
Assigned to you @adi271001
